manifests/run_service.pp

```diff
@@ -10,13 +10,18 @@
     default   => 'prometheus',
   }
 
+  $restart_cmd = $prometheus::init_style ? {
+    'systemd' => "systemctl reload-or-restart ${init_selector}",
```

Comment:
Does this work or should it be a fully qualified path?

Reply:
it works on debian jessie, the existing `manifests/config.pp` also calls systemctl unqualified.

Reply:
Then 👍 because I know the path can differ (```/bin/systemctl``` on Fedora and Jessie, ```/usr/bin/systemctl``` on Gentoo).